ast_convenience.mli

```diff
@@ -4,9 +4,9 @@
 
 (** {1 Convenience functions to help build and deconstruct AST fragments.} *)
 
-open Parsetree
 open Asttypes
 open Ast_helper
+open Parsetree   (* Open Parsetree after Asttypes since parsetree overrides constant type in 'ocaml 4.03' *)
```

Comment:
Wouldn't it be better to prefix the references to this constant type explicitly?